modules/mobile/app/workers/mobile/v0/vet360_linking_job.rb

```diff
@@ -25,10 +25,25 @@ class Vet360LinkingJob
       class MissingUserError < StandardError; end
 
       def perform(uuid)
-        user = IAMUser.find(uuid) || User.find(uuid)
+        user = IAMUser.find(uuid) || ::User.find(uuid)
```

Comment:
Now that I see this, I'm curious if we should name the model differently. It feels like a bug waiting to happen. I get that we have to have the `mobile` namespace. Will the constant loader still work if we call it `Mobile::MobileUser`? It's a clunky name, but at least it's clear and won't cause namespace confusion.